glacierscript.py

```diff
@@ -797,7 +800,7 @@ def withdraw_interactive():
                 print "{0} BTC going back to cold storage address {1}".format(value, address)
             else:
                 print "{0} BTC going to destination address {1}".format(value, address)
-        print "Fee amount: {0}".format(fee)
+        print "Fee amount: {0} btc ({1} mbtc)".format(fee,btc_to_mbtc(fee))
```

Comment:
DRY this up...two places where fee amount display is computed, and they are inconsistent (bitcoin/btc, mbtc/mBTC).

Reply:
I'm not sure exactly what you mean by this? I put this in because I just wanted an additional display in mBTC (in parentheses) to ensure I wasn't counting zeroes wrong in BTC. The only calculation/computation should be just multiplication by 1000 (not an actual full fee recalculation). Any clarification would be appreciated!

Reply:
Lines 477 and 803 (as of a0c9c6019a6ae5e122073f8a3bf466f59fe72d1e) both do the same thing, with subtle inconsistencies.

`{0} bitcoin = {1} mBTC`
`{0} btc ({1} mbtc)`

This formatting should be done in one place -- and might as well use it everywhere a BTC amount is displayed.

This will break the developer tests, so please include an update of those in the same commit.

Reply:
Understood - thank you for the clarification.